charts/netbird-dashboard/templates/deployment.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: {{ include "netbird-dashboard.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
```

Comment:
I want to be able to force the namespace, so my approach would be:
Define a helper
``` 
{{/*
Allow the release namespace to be overridden
*/}}
{{- define "netbird-dashboard.namespace" -}}
{{- default .Release.Namespace .Values.global.namespace -}}
{{- end -}}
```
And then define it in each template:
```
  namespace: {{ include "netbird-dashboard.namespace" . }}
```
Same thing for netbird helm chart